### ISSUE TITLE
Breaks existing routes inside express

### ISSUE THREAD

**Issue by mariorodriguespt** [wille/gatsby-plugin-express#2]
My use case: I've 2 repos, one for frontend and another to serve a REST API using express.
My goal is to server the static pages and the API from the same server.

I followed the instructions and made this scripts:
```
cd ../../Frontend
yarn build
cp -R public ../Backend
cp gatsby-express.json ../Backend/gatsby-express.json
```

Now I've the Gatbsy build inside my `/public` folder in the backend repo and also the `gatsby-express.json`.

If I use `app.use(express.static('public/'));`, it works as expected.

However with:
```
app.use(gatsyExpress('gatsby-express.json', {
    publicDir: 'public/'
  }));
```

The static pages work but it breaks my REST endpoints routes by showing the 404/Not Found page.

Am I missing something if I just use `app.use(express.static('public/'));`?

**Comment by piducancore** [comment]
I'm having this exact same issue trying to serve a gatsby (with apollo) builld from the same server that's running the graphql api.

The gatsby build is served correctly but the `/graphql` endpoint is not accessible while using ```app.use(gatsyExpress(...))```.
When commenting this, using only `app.use(express.static(path.join(__dirname, 'public/')))`, the gatsby build is served correctly and queries the api without problems.

I don't know what am I missing either. If someone could explain a little bit more about the use cases of this plugin, it would be great.

**Comment by wille** [comment]
fixed in 913a1abe93bc87cf0a8e6d20bad0d1486d20ddb3
